### DISCUSSION TITLE
[Feature request] add toaster/snackbar component

### DISCUSSION THREAD

**Discussion by hardikpthv** [ing-bank/lion#1274] [Open for contributions]
It would be very useful to have a toaster/snackbar component to be a part of the lion based on the existing overlays system. Are there already any plans for it? 🤔 

More info:
- https://www.w3.org/WAI/tutorials/forms/notifications/
- https://www.w3.org/TR/wai-aria-1.1/#alert

**Comment by lpoulter** [comment]
An Alert/Toast notification would be useful. Is it on the roadmap?

**Comment by vahidm90** [comment]
hey everyone... I have created something which is worth looking at! Lacks tests though. https://github.com/ing-bank/lion/pull/1729#issue-1268701066
